src/azure-cli/azure/cli/command_modules/backup/tests/latest/test_backup_commands.py

```diff
@@ -207,6 +209,7 @@ def test_backup_policy(self, resource_group, vault_name, policy1, policy2, vm1,
         ])
 
         self.kwargs['policy1_json']['name'] = self.kwargs['policy3']
+        self.kwargs['policy1_json']['properties']['instantRpDetails'] = {'azureBackupRgNamePrefix' : 'RG_prefix', 'azureBackupRgNameSuffix': 'RG_suffix'}
```

Comment:
Will this property be in the command result? If so, we should add a check for it.

Reply:
Added a check